From b524a9d56380dcedb55aadbf54862f77c5b17abd Mon Sep 17 00:00:00 2001
From: Patrice Chotard <patrice.chotard@foss.st.com>
Date: Thu, 28 Mar 2024 16:27:49 +0100
Subject: [PATCH] arm64: dts: st: Add button-user-2 on board stm32mp257f-ev1

Both button named USER1 and USER2 are available for Linux.

Change-Id: I74b9ae4fd9820540768497b961fa89fb142eb409
Signed-off-by: Patrice Chotard <patrice.chotard@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/370010
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 7 +++++++
 1 file changed, 7 insertions(+)

--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -72,6 +72,13 @@
 			status = "okay";
 		};
 
+		button-user-2 {
+			label = "User-2";
+			linux,code = <BTN_2>;
+			gpios = <&gpiog 8 GPIO_ACTIVE_HIGH>;
+			status = "okay";
+		};
+
 		button-wake-up {
 			label = "wake-up";
 			linux,code = <KEY_WAKEUP>;
